gobblin-api/src/main/java/org/apache/gobblin/configuration/ConfigurationKeys.java

```diff
@@ -117,6 +117,11 @@
   public static final String HELIX_JOB_TIMEOUT_SECONDS = "job.timeout.seconds";
   public static final String DEFAULT_HELIX_JOB_TIMEOUT_SECONDS = "10800";
 
+  public static final String HELIX_TASK_TIMEOUT_ENABLED_KEY = "task.timeout.enabled";
+  public static final boolean DEFAULT_HELIX_TASK_TIMEOUT_ENABLED = false;
+  public static final String HELIX_TASK_TIMEOUT_SECONDS = "task.timeout.seconds";
+  public static final long DEFAULT_HELIX_TASK_TIMEOUT_SECONDS = 60 * 60;
```

Comment:
The helix configuration is in milliseconds, so either make this milliseconds or multiply by 1000 when setting it.